commands.js

```diff
@@ -326,6 +326,8 @@ exports.commands = {
 		};
 		if (avatarid in avatarTable) {
 			avatar = avatarTable[avatarid];
+		} else if (avatarid === user.userid && user.confirmed === user.userid) {
+			avatar = '#' + avatarid;
```

Comment:
This also doesn't quite make sense - someone could do this who doesn't have a custom avatar globally and it would be a broken image...